modules/cluster/main.tf

```diff
@@ -66,6 +66,7 @@ module "eks" {
       asg_desired_capacity = var.desired_node_count
       asg_min_size         = var.min_node_count
       asg_max_size         = var.max_node_count
+      spot_price           = (var.enable_spot_instances ? var.spot_price : null)
```

Comment:
Need to define `enable_spot_instances` and `spot_price` in modules/cluster/variables.tf and then pass it as an input to main.tf.

Reply:
it is part of the commit already

Reply:
Done. Please check. Thanks!

Reply:
Cool :+1: 
Just need to squash (https://jenkins-x.io/community/code/#squash-and-rebase) the commits, and this should be good. What do you think about adding a small description to the readme (besides the terraform-docs output), as it will be clear that this module supports spot instances, wdyt @hferentschik @joebertj?

Reply:
Done squash and rebase. I can add a smal description to the readme.

Reply:
Tested it on my test cluster
![spot](https://user-images.githubusercontent.com/14118011/83935022-f355ec80-a7e7-11ea-8a76-37a198198870.PNG)